### PR TITLE
[Feat][BugFix]Support the Qwen3-Next-80B-A3B-Instruct quantization model&Fix the NZ issue

### PR DESCRIPTION
**Pull request by InSec**
### What this PR does / why we need it?
Support the Qwen3-Next-80B-A3B-Instruct quantization model and Fix the NZ issue. Triton kernel doesn't support data format nz, thus we skip converting weight to nz on layer `conv1d`

### Does this PR introduce _any_ user-facing change?
N/A
##### Installation steps
git clone https://gitcode.com/Ascend/msit.git
cd msit/msmodelslim
bash install.sh

#### Generate Qwen3-Next-80B-A3B-Instruct W8A8 weights
cd /example/Qwen3-Next
Command reference: [msmodelslim/example/Qwen3-Next/README.md](https://gitcode.com/Ascend/msit/blob/master/msmodelslim/example/Qwen3-Next/README.md) 

- vLLM version: v0.11.0
- vLLM main: https://github.com/vllm-project/vllm/commit/2918c1b49c88c29783c86f78d2c4221cb9622379

### CODE REVIEW

File: vllm_ascend/ops/linear.py [hunk 1]
```diff
@@ -45,8 +45,9 @@ class AscendUnquantizedLinearMethod(UnquantizedLinearMethod):
 
     def process_weights_after_loading(self, layer: torch.nn.Module) -> None:
         super().process_weights_after_loading(layer)
-        if (is_enable_nz() and layer.weight.data.dtype
-                in [torch.float16, torch.bfloat16]):
+        if "conv" not in layer.prefix and (is_enable_nz()
```
Comment (weijinqian0) [inline vllm_ascend/ops/linear.py]: why conv is not support nz

File: vllm_ascend/quantization/quant_config.py [hunk 2]
```diff
@@ -222,6 +222,8 @@ def get_scaled_act_names(self) -> List[str]:
         ],
         "gate_up_proj": ["gate_proj", "up_proj"],
         "in_proj": ["in_proj_qkvz", "in_proj_ba"],
+        "experts":
+        ["experts.0.gate_proj", "experts.0.up_proj", "experts.0.down_proj"]
```
Comment (weijinqian0) [inline vllm_ascend/quantization/quant_config.py:226]: This will support EP enable?

Comment (InSec) [reply vllm_ascend/quantization/quant_config.py:226]: yes

File: vllm_ascend/ops/linear.py (hunk 1 above)
Comment (InSec) [reply vllm_ascend/ops/linear.py]: The accuracy issue of the qwen3_next model is caused by converting 'conv' to NZ format.

File: tests/e2e/multicard/test_qwen3_next.py [hunk 3]
```diff
@@ -106,3 +112,22 @@ def test_models_distributed_Qwen3_NEXT_MTP_TP4_SIMILARITY():
...
+            snapshot_download(
+                "vllm-ascend/Qwen3-Next-80B-A3B-Instruct-W8A8-Pruning"),
+            max_model_len=4096,
+            tensor_parallel_size=2,
+            gpu_memory_utilization=0.4,
+            max_num_seqs=1,
+            enable_expert_parallel=True,
+            quantization="ascend",
+    ) as vllm_model:
+        vllm_model.generate_greedy(example_prompts, max_tokens)
```
Comment (MengqingCao) [inline tests/e2e/multicard/test_qwen3_next.py:134]: let's add `gold_results` to make sure the accuracy doesn't being broke

**Comment by MengqingCao** [comment]
Since the dependency of modelslim is changed, we need to update the doc on quatization feature. We can do this in next pr @InSec

**Review by MengqingCao — APPROVED** [review]
